specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/examples/AzureLargeInstance_Get.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "api-version": "2023-07-20-preview",
+    "subscriptionId": "f0f4887f-d13c-4943-a8ba-d7da28d2a3fd",
+    "resourceGroupName": "myResourceGroup",
+    "azureLargeInstanceName": "myAzureLargeInstance"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/f0f4887f-d13c-4943-a8ba-d7da28d2a3fd/resourceGroups/myResourceGroup/providers/Microsoft.AzureLargeInstance/AzureLargeInstance/myAzureLargeInstance",
+        "location": "westus2",
+        "name": "myAzureLargeInstance",
+        "tags": {
+          "key": "value"
+        },
+        "type": "Microsoft.AzureLargeInstance/AzureLargeInstance",
```

Comment:
>"type": "Microsoft.AzureLargeInstance/AzureLargeInstance", [](http://example.com/codeflow?start=8&length=58)

in the old days we would have called it a Microsoft.WindowsAzureLargeInstance/WindowsAzureLargeInstance...

just pointing out that putting brand in resource type names can date badly.

Reply:
Maybe just Microsoft.AzureLargeInstance/largeInstance would be incrementally better?
I also have plenty of curiosity about why this isn't couldn't be in some other provider namespace.

Reply:
For instance how does this API relate to /contrast to BareMetal or DedicatedHost APIs? Like
[https://learn.microsoft.com/en-us/rest/api/networkcloud/bare-metal-machines/get?tabs=HTTP](https://learn.microsoft.com/en-us/rest/api/networkcloud/bare-metal-machines/get?tabs=HTTP)

Reply:
or Microsoft.Compute for that matter?